app/templates/content/index.hbs

```diff
@@ -28,68 +28,187 @@
                         </ul>
                     </h5>
                     <h6 class='preprint-added-on'>
-                        {{t dateLabel}}: {{moment-format relevantDate "MMMM DD, YYYY"}} | {{t "content.header.last_edited"}}: {{moment-format model.dateModified "MMMM DD, YYYY"}}
+                        {{#if (not isWithdrawn)}}
+                            {{t dateLabel}}: {{moment-format relevantDate "MMMM DD, YYYY"}} | {{t "content.header.last_edited"}}: {{moment-format model.dateModified "MMMM DD, YYYY"}}
+                        {{else}}
+                            {{t dateLabel}}: {{moment-format relevantDate "MMMM DD, YYYY"}} | {{t "content.header.withdrawn_on"}}: {{moment-format model.dateWithdrawn "MMMM DD, YYYY"}}
+                        {{/if}}
                     </h6>
                 </div>
-                <div class="col-xs-12 col-sm-2 edit-button-and-logo text-center">
-                    {{#if isAdmin}}
-                        {{! TODO This link will need to be updated for phase 2 of branding }}
-                        <a type="button"
-                           class="btn btn-success edit-btn"
-                           href={{concat theme.guidPathPrefix model.id '/edit'}}
-                           onclick={{action "click" 'link' "Content - Edit Preprint" (concat model.links.html 'edit')}}
-                        >
-                            {{t editButtonLabel documentType=model.provider.documentType}}
-                        </a>
-                    {{/if}}
-                    <br>
-                    <div class="content-provider-logo provider-{{model.provider.id}} hidden-xs" title="{{model.provider.name}}"></div>
-                </div>
+                {{#if (not isWithdrawn)}}
+                    <div class="col-xs-12 col-sm-2 edit-button-and-logo text-center">
+                        {{#if isAdmin}}
+                            {{! TODO This link will need to be updated for phase 2 of branding }}
```

Comment:
Is there a ticket for this? If so please add it here so it doesn't get lost.

Reply:
That comment is part of the original template. See comment above.